packages/client/src/components/MovieCard/handlerFavorite.js

```diff
@@ -0,0 +1,57 @@
+import { apiURL } from '../../apiURL';
+
+export const handleAddFavorite = async (
+  movieID,
+  item,
+  favorites,
+  setFavorites,
+  userId,
+) => {
+  const body = {
+    movie_id: movieID,
+    user_id: userId,
+  };
+
+  try {
+    const response = await fetch(`${apiURL()}/favorites/`, {
+      method: 'POST',
+      body: JSON.stringify(body),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (response.status === 201) {
+      setFavorites([...favorites, item]);
+    }
+  } catch (error) {
+    return error;
+  }
+};
+
+export const handleRemoveFavorite = async (
+  movieId,
+  item,
+  favorites,
+  setFavorites,
+  userId,
+) => {
+  try {
+    const response = await fetch(`${apiURL()}/favorites/${movieId}/${userId}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    if (!response.ok) {
```

Comment:
I am not sure f this will work since there is already catch block that will capture if response will not be 2xx

Reply:
I have removed it nnow :)